### PR TITLE
new hero video

### PR DESCRIPTION
**Pull request by rileysklar**

### CODE REVIEW

**Review by rileysklar — COMMENTED** [review]
@riefer02 please review and approve

File: src/components/HeroMain.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default function HeroMain() {
         className="absolute z-0 hidden h-full w-full object-cover sm:block"
       >
         <source
-          src="https://tolbertscms.com/wp-content/uploads/2024/07/tolberts-hero.mp4"
+          src="http://tolbertscms.com/wp-content/uploads/2024/07/Copy-of-minute-cut.mp4"
```
Comment (riefer02) [inline src/components/HeroMain.jsx:16]: needs https @rileysklar

Comment (rileysklar) [reply src/components/HeroMain.jsx:16]: Good catch

Comment (riefer02) [reply src/components/HeroMain.jsx]: @rileysklar I just got it. Trying to remove those server side render files too

**Pull request merged**